charts/sysdig/README.md

```diff
@@ -148,165 +148,165 @@ The following table lists the configurable parameters of the Sysdig chart and th
+| `nodeAnalyzer.benchmarkRunner.image.repository`              | The image repository to pull the Benchmark Runner from.      | `sysdig/compliance-benchmark-runner`                                           |
+| `nodeAnalyzer.benchmarkRunner.image.tag`                     | The image tag to pull the Benchmark Runner.                  | `1.1.0.8`                                                                      |
+| `nodeAnalyzer.benchmarkRunner.image.digest`                  | The image digest to pull.                                    | ` `                                                                            |
+| `nodeAnalyzer.benchmarkRunner.image.pullPolicy`              | The Image pull policy for the Benchmark Runner.              | `IfNotPresent`                                                                 |
+| `nodeAnalyzer.benchmarkRunner.includeSensitivePermissions`   | Grants the service account elevated permissions to run CIS Benchmark for OS4. | `false`                                                                        |
+| `nodeAnalyzer.benchmarkRunner.resources.requests.cpu`        | Benchmark Runner CPU requests per node.                      | `150m`                                                                         |
+| `nodeAnalyzer.benchmarkRunner.resources.requests.memory`     | Benchmark Runner memory requests per node.                   | `128Mi`                                                                        |
+| `nodeAnalyzer.benchmarkRunner.resources.limits.cpu`          | Benchmark Runner CPU limit per node.                         | `500m`                                                                         |
+| `nodeAnalyzer.benchmarkRunner.resources.limits.memory`       | Benchmark Runner memory limit per node.                      | `256Mi`                                                                        |
+| `nodeAnalyzer.benchmarkRunner.env`                           | The extra environment variables that will be passed onto pods. | `{}`                                                                           |
+| `nodeAnalyzer.runtimeScanner.deploy`                         | Deploys the Runtime Scanner.                                 | `false`                                                                        |
+| `nodeAnalyzer.runtimeScanner.extraMounts`                    | Specifies a container engine custom socket path (docker, containerd, CRI-O). |                                                                                |
+| `nodeAnalyzer.runtimeScanner.image.repository`               | The image repository to pull the Runtime Scanner from.       | `sysdig/vuln-runtime-scanner`                                                  |
+| `nodeAnalyzer.runtimeScanner.image.tag`                      | The image tag to pull the Runtime Scanner.                   | `1.5.6`                                                                        |
```

Comment:
I believe there are extra spaces after `1.5.6` here that made the pre-commit generate the entire table again, would you mind double check if this is the case and possibly remove the extra spaces? 🙏

Reply:
same below for node-analyzer